pkgs/development/python-modules/pandas/default.nix

```diff
@@ -32,13 +37,13 @@ buildPythonPackage rec {
 
```

Comment:
Version looks wrong. So does the commit message.

Thanks github for not letting me comment on unchanged lines.

Reply:
Yes, it does.

Reply:
Commit message still wrong

Reply:
Thanks, I know. Title was also wrong.